apps/web/lib/components/kanban-card.tsx

```diff
@@ -216,32 +241,31 @@ export default function Item(props: ItemProps) {
 							</div>
 						)}
 					</div>
-					<CircularProgress percentage={10} />
+
+					<CircularProgress percentage={progress} />
 				</div>
 				<div className="my-2">
 					<HorizontalSeparator />
 				</div>
-				<div className="w-full flex items-center justify-between">
-					<div className="mt-1">
+				<div className="w-full h-10 flex items-center justify-between">
+					<div>
 						{item.status === TaskStatus.INPROGRESS ? (
 							<div className="flex items-center gap-2">
 								<small className="text-grey text-xs text-normal">Live:</small>
 								<p className="text-[#219653] font-medium text-sm">
-									{pad(hours)}:{pad(minutes)}:{pad(seconds)}{' '}
+									{h}h: {m}:m
```

Comment:
@Anishali2 I think here idea was to show "seconds" and with continues "refresh" it will look like "real-time" data etc. Even if we did not use yet such real-time updates, I would like to keep that idea. But correct me if I am wrong and this place makes sense only show minutes/hours, not seconds?

Reply:
Sir previously it shows gauzy-app-timer time which can render whole kanban after every milliseconds but now I've shown task total worked time..

Reply:
I've shown the seconds also

Reply:
@Anishali2 I don't get that part "previously it shows gauzy-app-timer time" but " now I've shown task total worked time", you mean you decide to show one value instead of another? Forget for a second about issue with "refresh" etc, I am asking logically you decided to show some DIFFERENT time? Explain please exactly what time was shown before and what time will be shown now from point of view of manager who overview that board data.